### PR TITLE
Implement SwiftTool.swift on Windows

### PR DESCRIPTION
**Pull request by jmittert**

### CODE REVIEW

File: Sources/Commands/SwiftTool.swift [hunk 1]
```diff
@@ -397,9 +398,15 @@ public class SwiftTool<Options: ToolOptions> {
                     to: sigaction.__Unnamed_union___sigaction_handler.self)
               #endif
                 sigaction(SIGINT, &action, nil)
+              #endif
 
+              #if os(Windows)
+                // Exit as if by signal()
+                TerminateProcess(GetCurrentProcess(), 3)
```
Comment (jmittert) [inline Sources/Commands/SwiftTool.swift]: [Apparently](https://docs.microsoft.com/en-us/previous-versions/xdkz3x12(v%3Dvs.140)), by default, on an uncaught `signal`, "By default, signal terminates the calling program with exit code 3, regardless of the value of sig."

File: Sources/Commands/SwiftTool.swift [hunk 2]
```diff
@@ -387,6 +387,7 @@ public class SwiftTool<Options: ToolOptions> {
                 // Terminate all processes on receiving an interrupt signal.
                 processSet.terminate()
 
+              #if !os(Windows)
```
Comment (aciidb0mb3r) [inline Sources/Commands/SwiftTool.swift]: Can you re-arrange this better? Maybe `#if os(Windows)` .... `#elseif os(macOS) .... #else`. Some amount of duplication is fine if it leads to a better code readability.

Comment (jmittert) [reply Sources/Commands/SwiftTool.swift]: Fair point, should be a bit more readable now.

**Comment by aciidb0mb3r** [comment]
@swift-ci smoke test

**Pull request merged**